doc/ideas/gameplay.md

```diff
@@ -188,16 +268,44 @@ Resources
 ### Infinite regeneration
 
  - Forest regeneration or seeding
- - Mills can produce sheep, cows, turkeys, etc.
+ - Animals reproduce as long as there are two of the same kind left on the whole map
 
 ### Stone balancing
 
-Stone cost for all buildings, rebalance stone amount per pile.
-Maybe new deep stone mines for later ages.
+ - Stone cost for all buildings, rebalance stone amount per pile.
+ - Maybe new deep stone mines for later ages.
+
+
+Relic
+---------
+
+ - Relic could have special abilities like in AOM
+  - e.g. they can have attack bonuses (patches) for special units or economic/military bonuses
+  - special abilities for every relic could either be generated when the map is generated or when the relic is discovered based on the actual needs of the player
+  - so the later you go out to get the relic the more it could get useful for you, because it could be better shaped on your personal military/economy but the risk is higher, that another player was going out before you
+  - if a player scouts the relic first and the ability gets generated in this moment, it will be for the player who scouted it first, so he knows, that this relic could help their own economy/military alot so the player will try to fight about this relic against the enemy heavily -> new gameplay aspect
 
 
+Production
+---------
+
+ - let the last produced unit in a buidling be produced for an infinite time (as long as ressources and pop space are available) --> AOM-style
+ - same for farms (either reseed a certain number with a waiting line or an option "reseed forever" as long as ressources are available)
+ - Mills can produce sheep, cows, turkeys, etc.
+
 For the lulz
 ------------
 
  - Big fat damage numbers!!!1
  - Friendly-fire toggle!!111
+
+
+Implement features the AoE-devs didn't do
+------------
+See here:
+https://de.scribd.com/document/318886164/AGE-2-Design-Document#browse-all-btn
```

Comment:
Since the document is probably still copyrighted, we shouldn't link to it here. Writing down the ideas from the docs should be fine though.